src/api/actions/checkBulkUploadJob.ts

```diff
@@ -0,0 +1,21 @@
+import { Maybe } from "../../utilities";
+import { UploadJobStatus } from "../../types";
+import { makeApiCall } from "./helpers";
+
+export const checkBulkUploadJob = async (
+  apiUri: string,
+  urls: string[]
```

Comment:
Checking jobs requires the job ids, not the urls, but the name of this parameter is urls

Reply:
corrected this